Makefile

```diff
@@ -18,11 +18,13 @@ clean_docs:
 # Remove any notebooks from docs
 clean_notebooks:
 	-rm -rf ./docs/example_notebooks/notebooks
+	-jupyter nbconvert --clear-output --inplace example-notebooks/binary-classifier/classifier_bin.ipynb
```

Comment:
does this command work with wildcards?   If so can we generalize, to "all" notebooksin example-notebooks or something similar.

Several other parts seem difficult to maintain if our library of examples expands